backend/graph/api/mutations.resolvers.go

```diff
@@ -451,6 +453,48 @@ func (r *mutationRootResolver) RemoveEntryFromMyList(ctx context.Context, entryI
 	return r.QueryRoot().MyList(ctx)
 }
 
+// UpdateUserMetadata is the resolver for the updateUserMetadata field.
+func (r *mutationRootResolver) UpdateUserMetadata(ctx context.Context, birthData model.BirthOptions, nameData model.NameOptions) (bool, error) {
+	ginCtx, err := utils.GinCtx(ctx)
+	if err != nil {
+		return false, err
+	}
+	u := user.GetFromCtx(ginCtx)
+	if u.IsRegistered() && !u.IsActiveBCC() {
+		displayName := nameData.First + " " + nameData.Last
+		_, err = r.AuthClient.UpdateUser(ctx, ginCtx.GetString(auth0.CtxUserID), auth0.UserInfo{
```

Comment:
Can we add some meta information here that identifies the user as a "BCC Media" subscriber?

Reply:
I thought that was going to be implemented by you guys or?